### PR TITLE
Add subfolder support to importantFiles and versionFiles to make.py

### PR DESCRIPTION
**Pull request by jonpas**
Will also copy non-English README.md files from documentation subfolder and version them just like the main English version.

### CODE REVIEW

**Comment by jonpas** [comment]
@ViperMaul ?

File: tools/make.py [hunk 1]
```diff
@@ -73,8 +73,8 @@
 prefix = "ace"
 pbo_name_prefix = "ace_"
 signature_blacklist = ["ace_server.pbo"]
-importantFiles = ["mod.cpp", "README.md", "AUTHORS.txt", "LICENSE", "logo_ace3_ca.paa"]
-versionFiles = ["README.md", "mod.cpp"]
+importantFiles = ["mod.cpp", "README.md", "documentation\\README_DE.md", "documentation\\README_PL.md", "AUTHORS.txt", "LICENSE", "logo_ace3_ca.paa"]
+versionFiles = ["README.md", "documentation\\README_DE.md", "documentation\\README_PL.md", "mod.cpp"]
```
Comment (jonpas) [inline tools/make.py]: Reminder: need to change location from `documentation` to `docs` when #2377 is merged.

**Comment by bux578** [comment]
If #2377 the path changes from "documentation" to "docs"

**Comment by jonpas** [comment]
Status?

**Pull request merged**